db/seeds.rb

```diff
@@ -1,7 +1,45 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'json'
+File.open("db/schools_data.json").each do |line|
+  parsed_course = JSON.parse(line) #Every line in a the .txt file represents an JSON course object with belonging time objects
+  department = parsed_course["department"]
+  grading_option = parsed_course["grading_option"]
+  # name = parsed_course["name"]
+  id = parsed_course["id"]
+  students_enrolled_max  = parsed_course["max_students"]
+  units= parsed_course["units"]
+  description= parsed_course["desc"]
+  instructor_name= parsed_course["instructors"] # returns instructors separated by comma
+  is_graduate_course = parsed_course['is_grad']
+  # Unused data: students_enrolled_actually, prequisites
+  if instructor_name.include? ","
+    instructor_name = instructor_name.split(",")[0]
+  end
+  instructor = Instructor.where(id: instructor_name).first_or_create(rmp_url: "lol")
+  course = instructor.courses.where(dept: department, course_no: id).first_or_create(description: description,
```

Comment:
Reformat so lines are 80 characters or less. It's hard to read the code when you split your editor to view multiple files